rmc/utils/mpc.py

```diff
@@ -302,9 +280,8 @@ def create_context_with_oe(
     ht = ht.select(oe_col).key_by("locus", "alleles")
     ht = ht.collect_by_key()
     ht = ht.annotate(**{oe_col: hl.nanmin(ht.values[oe_col])})
-    transcript_col = "transcript_rescue" if include_rescue else "transcript"
     ht = ht.annotate(
-        **{transcript_col: ht.values.find(lambda x: x[oe_col] == ht[oe_col]).transcript}
+        **{"transcript": ht.values.find(lambda x: x[oe_col] == ht[oe_col]).transcript}
```

Comment:
```suggestion
    ht = ht.select("oe").key_by("locus", "alleles")
    ht = ht.collect_by_key()
    ht = ht.annotate(**{"oe": hl.nanmin(ht.values["oe"])})
    ht = ht.annotate(
        **{"transcript": ht.values.find(lambda x: x["oe"] == ht["oe"]).transcript}
    )

Reply:
am I missing something? this looks right, right?
<img width="844" alt="image" src="https://user-images.githubusercontent.com/19142091/216075598-73a99c70-3d50-47f8-a28a-a360a76bd4e6.png">

Reply:
ohh, oe_col. will fix